package.json

```diff
@@ -26,6 +26,10 @@
   "bugs": {
     "url": "https://github.com/mmw/react-password-strength/issues"
   },
+  "engines": {
```

Comment:
This is way too aggressive, the package works on older versions of node/npm. Either remove or I need to figure out what is a reasonable setting for this